### DISCUSSION TITLE
How to display log in date range (from date to date) from UI

### DISCUSSION THREAD

**Discussion by doaortu** [logdyhq/logdy-core#47] [Q&A] [answered]
Hi, I've known and read that there's faceted filter, but I don't know to use it to display log by date range.

Can logdy do something like that? or is it limited to the command modes (`cut-by-date`)?

Basically I'm seeking something like `cut-by-date` but in the UI. 

Thanks

**Comment by PeterOsinski** [comment, marked as answer]
Hi doaortu, currently its not supported as it would require indexing the dates. I'm working on something that will satisfy that requirement but it's still a bit long way to go and release. Current version of Logdy is intended for tailing logs mostly and its not a great fit to browse large amounts of logs (that purpose to satisfy is on the roadmap)

**Reply by doaortu** [reply]
Got it, thank you for the confirmation, I'll look for the workaround for now.
